_lib/wordpress_view_processor.py

```diff
@@ -51,8 +51,9 @@ def process_view(post):
         hero_url = os.path.expandvars("$WORDPRESS/hero/" + hero_id + "/?json=1")
         response = requests.get(hero_url)
         hero_data = json.loads(response.content)
-        hero_data = hero_data['post']
-        hero_data['related_posts'] = hero_data['custom_fields']['related_post']
-        post['hero'] = hero_data
+        if hero_data['status'] is "ok":
```

Comment:
Whatever convention you'll have (if any) on the python-side is fine. In the JS we're normalized on single quotes, in the HTML double. Sounds like it makes sense to have it follow the output.